File: 20220928/모범답안/20220928_01.js/repeatAlphabet.js
function solution(S)
{
    var arr = [];

    for(var i = 0; i < S.length; i++){
        if(arr[arr.length - 1] == S[i]){
            arr.pop();
            continue;
        }

        arr.push(S[i]);
    }

    if(arr.length == 0){
        return 1;
    }

    return 0;
}

File: 20220928/모범답안/20220928_01.js/repeatAlphabet.py
def repeatRemove(S):
    if S == "":
        return ""
    else:
        return re.sub('(([a-zA-Z])\\2{1,})', '', S)


def solution(S):
    retStr = None

    while True:
        retStr = repeatRemove(S)
        if retStr == S or retStr == "":
            break
        else:
            S = retStr

    if retStr == "":
        return 1
    else:
        return 0